Mage.Sets/src/mage/cards/l/LocalizedDestruction.java

```diff
@@ -76,14 +76,20 @@ public boolean apply(Game game, Ability source) {
         if (controller == null) {
             return false;
         }
-        if (!controller.chooseUse(this.getOutcome(),
-        "Pay 1 or more {E}? Each creature you control with power equal to the amount of paid this way gains indestructible until end of turn", source, game)) {
+
+        int totalEnergy = controller.getCountersCount(CounterType.ENERGY);
+        if (totalEnergy == 0) {
             return false;
         }
-        int numberToPay = controller.getAmount(1, controller.getCountersCount(CounterType.ENERGY),
+        int numberToPay = controller.getAmount(1, totalEnergy,
```

Comment:
You should still ask for if any energy wants to be paid before asking the amount.

Reply:
oops didn't mean to change the order